Podfile

```diff
@@ -2,7 +2,7 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 use_frameworks!
 
-pod 'IQKeyboardManagerSwift', '~>3.3.7'
+pod 'IQKeyboardManagerSwift'
```

Comment:
Enforcing the version is always a good idea. We can be loose and just say 4.0 if we're not picky.